application/src/main/java/org/opentripplanner/graph_builder/module/OsmBoardingLocationsModule.java

```diff
@@ -99,14 +113,14 @@ public void buildGraph() {
   }
 
   private boolean connectVertexToStop(TransitStopVertex ts, StreetIndex index) {
```

Comment:
This method is getting too large. Please extract three methods for the three cases:

1. Node
2. Area
3. Way

Move the inline comments into Javadoc.